components/sbm-recipes-mule-to-boot/src/main/java/org/springframework/sbm/mule/actions/javadsl/translators/DslSnippet.java

```diff
@@ -59,4 +62,14 @@ public DslSnippet(String renderedSnippet,
                       Set<Bean> beans) {
         this(renderedSnippet, requiredImports, requiredDependencies, beans, false);
     }
+
+
+    public static String getMethodParameters(List<DslSnippet> dslSnippets) {
+        return dslSnippets.stream()
+                .flatMap(dsl -> dsl.getBeans().stream())
+                .collect(Collectors.toSet())
+                .stream()
+                .map(b -> b.getBeanClass() + " " + Helper.sanitizeForBeanMethodName(b.getBeanName()))
```

Comment:
Can we now move call to Helper into `Bean.getbeanName()` :) 
This will always be called from clients otherwise, or?

Reply:
I dont think we even need Helper. sanitizeForBeanMethodName() here, the Bean.getName() should always return a sane value because we are constructing it in DSL snippets which will always be hardcoded never user generated.